docs/source/markdown/podman-stats.1.md

```diff
@@ -98,6 +98,10 @@ ID             NAME           MEM USAGE / LIMIT
 6eae9e25a564   clever_bassi   3.031MB / 16.7GB
 ```
 
+Note: When using a slirp4netns network, the traffic send via the port forwarding will be accounted
```

Comment:
you should mention the rootlesskit port handler, with `--network slirp4netns:port_handler=slirp4netns` it should be counted correctly on tap0 AFAIK